03_QuestionAnswering/02_rag_claude_titan_pinecone.ipynb

```diff
@@ -564,30 +591,22 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Now we have answered the question being asked but also included the source of this information being used by the LLM.\n",
+    "Now we have answered the question being asked but also included the source of this information being used by the LLM.\n",
```

Comment:
qa_with_sources(query) returned the following for me where the sources is empty.

`{'question': 'How has AWS evolved?',
 'answer': " Based on the provided excerpts, it seems AWS has evolved significantly over time. Originally it started as a simple cloud computing service, but over time it has grown into a much more expansive offering. The excerpts mention how in the early days people questioned whether compute would be an undifferentiated commodity, but AWS has become a major differentiator for Amazon. It has also helped enable business continuity during the COVID-19 pandemic as companies shifted to remote work. The excerpt from the 2021 shareholder letter notes how AWS and Amazon's consumer business have had different demand trajectories during the pandemic, indicating AWS's growth and evolution into a multifaceted cloud platform.\n\n",
 'sources': ''}`